src/main/splitting/python_task_checker.py

```diff
@@ -0,0 +1,44 @@
+import os
+import sys
+import logging
+
+from src.main.util import consts
+from src.main.util.consts import LANGUAGE
+from src.main.util.file_util import create_file
+from src.main.util.language_util import get_extension_by_language
+from src.main.splitting.task_checker import ITaskChecker, check_call_safely, check_output_safely, TASKS_TESTS_PATH, \
+    SOURCE_OBJECT_NAME
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+class PythonTaskChecker(ITaskChecker):
+
+    @property
+    def language(self):
+        return LANGUAGE.PYTHON.value
+
+    # a=int(input())
+    # print(a)
+    @property
+    def min_symbols_number(self):
+        return 20
+
+    @property
+    def output_strings(self):
+        return ['print']
+
+    def create_source_file(self, source_code: str):
+        source_code_file = os.path.join(TASKS_TESTS_PATH, SOURCE_OBJECT_NAME,
```

Comment:
We have the same the code fragment in all checkers (we create source_code_file path and after to run create_file). I think it would be better to put it in a separate static method and run in each class with necessary parameters